docs/src/terminology.md

```diff
@@ -68,6 +68,14 @@ See [vote credit](terminology.md#vote-credit).
 
 A multicast network used to efficiently validate [entries](terminology.md#entry) and gain consensus.
 
+## delinquent
+
+When a validator does not submit a vote, it is known as delinquent. It is measured in the amount of blocks since a vote was last seen from that validator.
```

Comment:
How about, "The state of a staked validator when it is not voting."

We say things like, "a validator has been delinquent for X blocks." Delinquency is a state, not a measurement. I think you can delete that last sentence.